public/index.html

```diff
@@ -24,7 +24,8 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-    <title>React App</title>
+    <title>MentorPaired</title>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
```

Comment:
Instead of loading all font-awesome icon fonts here, you could make use of the [font-awesome react package](https://github.com/FortAwesome/react-fontawesome). It would ensure you don't have bloated builds.

Reply:
okay, will do.